tests/helper/abcHelper.ts

```diff
@@ -635,13 +639,14 @@ const mdFormAbc = (): ITestJson => ({
           pageIdentifier: '5',
           type: 'input',
           title: 'Remaining Issues',
+          hintText: 'Please describe the issue',
           subtitle: 'Please provide details about remaining issue',
           nextPageIdentifier: '6',
           options: {
             required: true,
             inputType: 'multiline',
             maximumCharacterCount: 300,
-            keyboardType: 'UIKeyboardTypeEmailAddress',
+            keyboardType: 'UIKeyboardTypeTwitter',
```

Comment:
can this change cause errors if brands use the old "keyboardType" in their requests instead of the new one?

Reply:
This entire functionally is new and not deployed anywhere, also i have tested and added enum for keyboardType for the values which are working.